menu/additional_apps.sh

```diff
@@ -34,6 +33,11 @@ choice=$(whiptail --title "Which apps do you want to install?" --checklist "Auto
 "SMB-mount" "(Connect to SMB-shares from your local network)" OFF 3>&1 1>&2 2>&3)
 
 case "$choice" in
+    *"Documentserver"*)
+        clear
+        print_text_in_color "$ICyan" "Downloading Documentserver.sh..."
```

Comment:
As I mentioned before, I think it looks better if the info says "Downloading the X script..."

Please replace in all places.

Reply:
done